fe/src/main/java/org/apache/doris/catalog/DynamicPartitionProperty.java

```diff
@@ -46,15 +48,14 @@
     private int buckets;
     private StartOfDate startOfWeek;
     private StartOfDate startOfMonth;
-    // TODO: support setting timezone.
     private TimeZone tz = TimeUtils.getDefaultTimeZone();
 
-
     public DynamicPartitionProperty(Map<String, String> properties) {
         if (properties != null && !properties.isEmpty()) {
             this.exist = true;
             this.enable = Boolean.parseBoolean(properties.get(ENABLE));
             this.timeUnit = properties.get(TIME_UNIT);
+            this.tz = TimeZone.getTimeZone(properties.getOrDefault(TIME_ZONE, ZoneId.systemDefault().toString()));
```

Comment:
Better move the time related functions to TimeUtils class

Reply:
done